src/http.rs

```diff
@@ -99,6 +102,42 @@ impl HttpClient {
         })
     }
 
+    /// Function `entity_timed` is like `entity` but with two optional fields `transaction_time` and `valid_time` that are of type `Option<DateTime<FixedOffset>>`.
+    pub fn entity_timed(
+        &self,
+        id: String,
+        transaction_time: Option<DateTime<FixedOffset>>,
+        valid_time: Option<DateTime<FixedOffset>>,
+    ) -> Result<Edn, CruxError> {
+        if !id.starts_with(":") {
+            return Ok(edn!({:status ":bad-request", :message "ID required", :code 400}));
+        }
+
+        let mut s = String::new();
+        s.push_str("{:eid ");
```

Comment:
Is it `:eid` though? I am searching on the docs and I can't find this keyword being used

Reply:
yes hahahahah, in the docker hub it was explained 😂😂😂